### ISSUE TITLE
Nether spigot and server.properties reference non-existant variables

### ISSUE THREAD

**Issue by wingzero54** [DevotedMC/Rhea-Public#23]
nether/server.properties references { world_seed } which is not defined anywhere
nether/spigot.yml references a bunch of seeds which are likewise not defined anywhere

**Comment by Diet-Cola** [comment]
Makes sense, these variables should be under a private folder that currently doesnt work on Rhea, however we still manually add these in on Dev Hell
